pydantic_factories/fields.py

```diff
@@ -17,6 +17,26 @@ def to_value(self) -> Any:
         return self.cb(*self.args, **self.defaults)
 
 
+class PostGenerated:
+    """
+    A class to allow post generating using already generated values.
+    The callback will be invoked after building all non post generated
+    factory attributes.
+
+    Callback should be able to receive the field name as its first argument
+    and a dictionary of values as it second argument.
+    """
+
+    def __init__(self, cb: Callable, *args: Any, **defaults: Any) -> None:
+        self.cb = cb
+        self.defaults = defaults
+        self.args = args
+
+    def to_value(self, name: str, values: dict[str, Any]) -> Any:
```

Comment:
According to test failure, I think you'll need to use Typing.Dict to be compatible < 3.9